proto/attestation/verification.proto

```diff
@@ -155,9 +155,6 @@ message KernelLayerData {
 message ApplicationLayerData {
   // Measurement RawDigest of the application binary.
   RawDigest binary = 1;
-
-  // Measurement RawDigest of the application configuration.
-  RawDigest config = 2;
```

Comment:
If you're removing the config reference values, should the config endorsements be removed as well?

https://github.com/project-oak/oak/blob/main/proto/attestation/endorsement.proto#L69